ios/Video/RCTVideo.swift

```diff
@@ -8,39 +8,39 @@ import React
 import Promises
 
 class RCTVideo: UIView, RCTVideoPlayerViewControllerDelegate, RCTPlayerObserverHandler {
-
+    
     private var _player:AVPlayer?
     private var _playerItem:AVPlayerItem?
     private var _source:VideoSource?
     private var _playerBufferEmpty:Bool = true
     private var _playerLayer:AVPlayerLayer?
     private var _chapters:[Chapter]?
-
+    
     private var _playerViewController:RCTVideoPlayerViewController?
     private var _videoURL:NSURL?
-
+    
```

Comment:
can you please clean up unnecessary spacing in this file please ?

Reply:
Yes I can, but I would love to setup Swift formater to avoid this in feature - so I will do it tomorrow and use it in this PR. manually is to much work if is blocking for you let's hold this PR for now

Reply:
@KrzysztofMoch gentle reminder ! :)

Reply:
Done ✅